README.md

```diff
@@ -42,31 +42,36 @@ torii: {
 
 Add the [acmidm-login-service](http://github.com/lblod/acmidm-login-service) in the backend to provide the necessary API endpoints.
 
-Configure authentication with [ember-simple-auth](https://github.com/simplabs/ember-simple-auth) and put the `{{acmidm-login}}` and `{{acmidm-logout}}` components on the appropriate pages. They will handle authentication with ACM/IDM automatically.
+Configure authentication with [ember-simple-auth](https://github.com/simplabs/ember-simple-auth) and put the `<Acmidm::Login>` component on the appropriate pages. This will handle authentication with ACM/IDM automatically. 
 
-Finally, overwrite the `sessionInvalidated` event handler of Ember Simple Auth's `application-route-mixin`:
+> The `<Acmidm::Login>` component does not output any HTML so your project will need to provide this. A usage example can be found [in the dummy app](https://github.com/lblod/ember-acmidm-login/blob/91bcd31655b27b51dce47ed25b67a64d7a15049b/tests/dummy/app/templates/application.hbs#L14-L31).
+
+Finally, extend Ember Simple Auth's `SessionService` and override the `handleInvalidation` method:
 
 ```javascript
-import Route from '@ember/routing/route';
-import ApplicationRouteMixin from 'ember-simple-auth/mixins/application-route-mixin';
-import ENV from 'frontend-loket/config/environment';
+import { inject as service } from '@ember/service';
+import BaseSessionService from 'ember-simple-auth/services/session';
+import ENV from 'app-name/config/environment';
+
+export default class SessionService extends BaseSessionService {
+  @service currentSession;
 
-export default Route.extend(ApplicationRouteMixin, {
-  sessionInvalidated() {
+  handleInvalidation() {
     const logoutUrl = ENV['torii']['providers']['acmidm-oauth2']['logoutUrl'];
-    window.location.replace(logoutUrl);
+    super.handleInvalidation(logoutUrl);
   }
 }
 ```
 
-To support switching without doing a full logout, set the appriopriate switchUrl in the torii configuration, add a `{{acmidm-switch}}` component and set up a switch route. This route should trigger a login, for example:
+To support switching without doing a full logout, set the appriopriate `switchUrl` in the torii configuration, add a `<Acmidm::Switch>` component and set up a switch route. This route should trigger a login, for example:
 
 ```javascript
-import UnauthenticatedRouteMixin from 'ember-simple-auth/mixins/unauthenticated-route-mixin';
 import Route from '@ember/routing/route';
 import { inject as service } from '@ember/service';
-export default Route.extend(UnauthenticatedRouteMixin, {
```

Comment:
I didn't include the `UnauthenticatedRouteMixin` replacement in the updated snippet since it didn't seem relevant for the switch example, but I can add it if needed.

Reply:
doesn't seem necessary indeed